nodes/analyzer/raycaster_lite.py

```diff
@@ -30,7 +30,8 @@ class SvRaycasterLiteNode(bpy.types.Node, SverchCustomTreeNode):
     bl_label = 'Raycaster'
     bl_icon = 'OUTLINER_OB_EMPTY'
     sv_icon = 'SV_RAYCASTER'
-
+    
+    MaxDistance: bpy.props.FloatProperty(name='MaxDistance', description='MaxDistance', default=1.0, update=updateNode)
```

Comment:
```suggestion
    max_distance: bpy.props.FloatProperty(name='MaxDistance', description='MaxDistance', default=1.0, update=updateNode)
```

Reply:
max_distance Variable already exists

Reply:
`max_distance` or `MaxDistance` is not a variable it is an attribute of the node class. Also this class has `start`, `direction` and no more other attributes. According Python convention names of a class should be in snake case. Sverchok code is not totally written according to this convention but it's better always to do for new code.